stories/components.stories.js

```diff
@@ -363,3 +364,24 @@ storiesOf("Components/Filter", module)
       );
     })
   );
+
+storiesOf("Components/Copyright,module").add("Copyright", () => {
```

Comment:
Should this be in Components/Data Display? cc @nerdlet

Reply:
@gertie-sheshe @KobbyMmo Yap. Everything that involved data or content should be in `Component/Data Display` storyof.